pvlib/test/test_singlediode_methods.py

```diff
@@ -125,3 +126,135 @@ def test_brentq_fs_495():
                                 method='lambertw')
     assert np.isclose(pvs_ixx, ixx)
     return isc, voc, imp, vmp, pmp, i, v, pvs
+
+
+def pvsyst_fs_495():
+    """
+    PVSyst First Solar FS-495 parameters.
```

Comment:
`PVsyst applied to First Solar FS-495 parameters.`

Reply:
✔️ the initial data comes from the PVSyst v6.7.2 database, I don't think any of the values were altered but it was vetted internally.

I've removed the method, and hard coded the values with a comment re: initial source.

Reply:
Not having a current PVsyst license, I'm guessing that it does NOT permit uploading that database to pvlib?

Reply:
Maybe not the whole database, but I'm guessing the relevant values come from pan files generated by FS.